programs/zstdless

```diff
@@ -1,2 +1,6 @@
 #!/bin/sh
-zstdcat "$@" | less
+
+# TODO: Address quirks and bugs tied to old versions of less, provide a mechanism to pass flags directly to zstd
+
+export LESSOPEN="|zstd -cdfq %s"
+less "$@"
```

Comment:
Also potentially we could use the `exec` command here (`exec less "$@"`), which would save us a process creation.